plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3630,6 +3634,38 @@ public void testEmptyDelete()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCtasFromAVersionedTable()
+            throws Exception
+    {
+        String sourceTableName = "test_ctas_versioned_source_" + randomTableSuffix();
+        String sinkTableNameVersion = "test_ctas_versioned_sink_" + randomTableSuffix();
+        String sinkTableNameTimestamp = "test_ctas_versioned_sink_" + randomTableSuffix();
```

Comment:
Maybe `sinkTableNameSnapshotVersioned` and `sinkTableNameTimestampVersioned`

Reply:
Changed to `snapshotVersionedSinkTableName` and `timestampVersionedSinkTableName`